<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_addnode</class>
 <widget class="QDialog" name="Dialog_addnode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>258</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add new node</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0,0,0,0,0,0,0">
     <property name="horizontalSpacing">
      <number>6</number>
     </property>
     <item row="9" column="1">
      <widget class="QLineEdit" name="lineEdit_username">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="9" column="2">
      <widget class="QLineEdit" name="lineEdit_password">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="clearButtonEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="3">
      <widget class="QRadioButton" name="radioButton_addscratch">
       <property name="text">
        <string>Add form scratch</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="3">
      <widget class="QLineEdit" name="lineEdit_name"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Protocol</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QPushButton" name="pushButton_discard">
       <property name="text">
        <string>Discard</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QCheckBox" name="checkBox_auth">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Auth</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QLineEdit" name="lineEdit_port">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::PreventContextMenu</enum>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhNone</set>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QComboBox" name="comboBox_protocol">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>ws</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ws+ssl</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>http</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>http+ssl</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>socks4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>socks5</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Host</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="3">
      <widget class="QRadioButton" name="radioButton_addcommand">
       <property name="text">
        <string>Add from command</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="3">
      <widget class="QLineEdit" name="lineEdit_command">
       <property name="enabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="lineEdit_host">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Username</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Port</string>
       </property>
      </widget>
     </item>
     <item row="10" column="2">
      <widget class="QPushButton" name="pushButton_save">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="3">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>radioButton_addscratch</sender>
   <signal>toggled(bool)</signal>
   <receiver>comboBox_protocol</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>191</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>49</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_addcommand</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineEdit_command</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>147</x>
     <y>16</y>
    </hint>
    <hint type="destinationlabel">
     <x>150</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_addscratch</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineEdit_host</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>191</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>163</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_addscratch</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineEdit_port</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>191</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_addscratch</sender>
   <signal>toggled(bool)</signal>
   <receiver>checkBox_auth</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>191</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>49</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_auth</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineEdit_username</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>49</x>
     <y>194</y>
    </hint>
    <hint type="destinationlabel">
     <x>163</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_auth</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineEdit_password</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>49</x>
     <y>194</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_discard</sender>
   <signal>clicked(bool)</signal>
   <receiver>Dialog_addnode</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>167</x>
     <y>234</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_save</sender>
   <signal>clicked(bool)</signal>
   <receiver>Dialog_addnode</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>307</x>
     <y>234</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
